openupgrade_scripts/scripts/base/18.0.1.3/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,220 @@
+base         / res.partner.bank         / country_code (char)           : previously in module account_qr_code_emv
+base         / res.users                / device_ids (one2many)         : NEW relation: res.device
+
+# NOTHING TO DO
+
+---XML records in module 'base'---
+NEW ir.actions.act_window: base.action_user_device
+NEW ir.actions.act_window: base.ir_embedded_action
+DEL ir.actions.act_window: base.ir_property_form
+NEW ir.config_parameter: base.default_max_email_size (noupdate)
+NEW ir.model.access: base.access_ir_cron_progress_group_cron
+NEW ir.model.access: base.access_ir_embedded_actions_group_user
+NEW ir.model.access: base.access_res_device
+NEW ir.model.access: base.access_res_device_log
+DEL ir.model.access: base.access_ir_property_group_system
+DEL ir.model.access: base.access_ir_property_group_user
+DEL ir.model.access: base.access_res_config_installer
+NEW ir.model.constraint: base.constraint_ir_act_client_path_unique
+NEW ir.model.constraint: base.constraint_ir_act_report_xml_path_unique
+NEW ir.model.constraint: base.constraint_ir_act_server_path_unique
+NEW ir.model.constraint: base.constraint_ir_act_url_path_unique
+NEW ir.model.constraint: base.constraint_ir_act_window_path_unique
+NEW ir.model.constraint: base.constraint_ir_actions_path_unique
+NEW ir.model.constraint: base.constraint_ir_cron_check_strictly_positive_interval
+NEW ir.model.constraint: base.constraint_ir_embedded_actions_check_only_one_action_defined
+NEW ir.model.constraint: base.constraint_ir_embedded_actions_check_python_method_requires_name
+NEW ir.model.constraint: base.constraint_ir_filters_check_res_id_only_when_embedded_action
+NEW ir.model.constraint: base.constraint_ir_filters_check_sort_json
+ir.model.constraint: base.constraint_ir_filters_name_model_uid_unique (changed definition: is now 'unique(model_id,user_id,action_id,embedded_action_id,embedded_parent_res_id,name)' ('unique(model_id,user_id,action_id,name)'))
+NEW ir.model.constraint: base.constraint_ir_mail_server_certificate_requires_tls
+NEW ir.model.constraint: base.constraint_res_groups_check_api_key_duration
+NEW ir.module.module: base.module_accountant (noupdate)
+DEL ir.module.module: base.module_account_accountant (noupdate)
```

Comment:
xmlid_rename?

Reply:
added to renamed_modules in apriori.py, but probably we'll never encounter that as this and below are enterprise modules